auth_mellon_config.c

```diff
@@ -1593,6 +1593,13 @@ const command_rec auth_mellon_commands[] = {
         "A list of AuthnContextClassRef to request in the AuthnRequest and "
         "to validate upon reception of an Assertion"
         ),
+    AP_INIT_TAKE1(
+        "MellonAuthnContextComparisonType",
+        ap_set_string_slot,
```

Comment:
Since this attribute is constrained to be one of 4 possible string values you must validate the incoming value in a custom function. There are several examples of these custom "setters" near the top of auth_mellon_config.c, all of them end with "_slot".